sdk/src/main/kotlin/io/rover/sdk/services/MeasurementService.kt

```diff
@@ -22,7 +22,11 @@ internal class MeasurementService(
     private val barcodeRenderingService: BarcodeRenderingService
 ) {
 
-    @SuppressLint("NewApi")
+    // Roundtrip to avoid rounding causing mismatched when converting floats to ints
+    fun measureDpToPxToDp(dpValue: Int): Float {
+        return dpValue.dpAsPx(displayMetrics).pxAsDp(displayMetrics)
+    }
```

Comment:
This seems odd to me.  Is this trying to avoid a rounding error further down the line as values are converted back and forth?

Reply:
```kotlin
// Snap the given Dp value to the nearest pixel, returning the equivalent Dp value.  The goal
// here is to ensure when the resulting Dp value is converted to a Px value (for the same
// density), no rounding will need to occur to yield an exact Px value (since Px values cannot be fractional).  This is useful because we do not want to accrue rounding errors downstream.
    fun snapToPixValue(dpValue: Int): Float {
        return dpValue.dpAsPx(displayMetrics).pxAsDp(displayMetrics)
    }
```